persistence/mongo/src/main/scala/za/co/absa/spline/persistence/mongo/dao/LineageDAOv4.scala

```diff
@@ -172,6 +173,10 @@ object LineageDAOv4 {
     val values: Seq[SubComponent] = sealedInstancesOf[SubComponentV4].toSeq
   }
 
+  def enforceDefaultMetricsOnWriteOperation(writeOp: DBObject): Unit = {
```

Comment:
Doesn't this method duplicate the one from `LineageDAOv4` class? https://github.com/AbsaOSS/spline/blob/ec001ec0a383f333cb269b867488038f0ee24991/persistence/mongo/src/main/scala/za/co/absa/spline/persistence/mongo/dao/LineageDAOv4.scala#L97

Reply:
Of course, there should be only one method. The duplication was caused by merge.